### PR TITLE
base commit to task tracker for issue #84

### PR DESCRIPTION
**Pull request by ThanaReka**
Completing the PR template for issue 84.

## TODO: Link the issue # to this PR
* Implements issue #84 

## TODO: Request a maintainer to review your PR
* @renweview @gaurigadkari  My PR is ready for review!

## TODO: Write a brief description of what your PR accomplished
* How did your PR fix the issue mentioned above? What files were changed or created? What did your changes do?
* A list screen has been created with headers, a + button, and containers for list items.
* I also added task items with hardcoded string values for now so that I can see visually what the layout looks like while addressing this issue.
* Files changed were: TaskListScreen.kt, TaskItemView.kt, TaskRepository.kt, Task.kt, and strings. xml.  The dimens.xml file was created while addressing this issue. 

## TODO: Attach screenshots of your changes
Screenshot_20240203_230728

### CODE REVIEW

File: coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/ui/TaskList/TaskListScreen.kt [hunk 1]
```diff
@@ -1,7 +1,172 @@
...
+
+}
+
+
+
+
+
+
+
+
```
Comment (gaurigadkari) [inline coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/ui/TaskList/TaskListScreen.kt:172]: can we remove these extra lines

File: coding-projects/android/TaskTracker/app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="content_padding">16dp</dimen>
+    <dimen name="task_list_padding">15dp</dimen>
```
Comment (gaurigadkari) [inline coding-projects/android/TaskTracker/app/src/main/res/values/dimens.xml:5]: as per the material design guidelines it is better to have multiples of 8 as dimens and for smaller multiples of 4 and lastly multiples of 2

File: coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/data/model/Task.kt [hunk 3]
```diff
@@ -6,4 +6,7 @@ import androidx.room.Entity
  * Created by Gauri Gadkari on 1/23/24.
  */
 @Entity
-data class Task(val taskTitle: String)
+data class Task(val taskTitle: String, val taskTime: String, val date: String)
+
+
+
```
Comment (gaurigadkari) [inline coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/data/model/Task.kt:12]: can we remove extra lines

**Comment by vuinguyen** [comment]
@renweview @gaurigadkari 
I got this screenshot in the Compose preview window. Also, it looks like the approved changes have been made.

Screenshot 2024-02-08 at 12 28 22 PM

**Pull request merged**